Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -167,7 +166,8 @@ private extension AppSettingsStore {
 
     func loadInAppFeedbackCardVisibility(onCompletion: (Result<Bool, Error>) -> Void) {
```

Comment:
> I'm trying to visualize how we would use this for both the general and product feedback. We would probably be passing a String in here, right?

I was planning to have different methods 
```
loadInAppFeedbackCardVisibility
loadProductsFeedbackBannerVisibility
```

As we don't know how future surveys will be presented. 🤔 

> Or perhaps an enum

At the moment, I had only plain string constants like `generalFeedbackName` or `productsFeedbackName` as I wasn't planning to switch over the enum and I wasn't sure where to put it 😅, What do you think? It can also be changed easily in the future if needed

Reply:
I think I'd prefer just having a single action. It's less costly to maintain, I guess?

Reply:
So something like `loadFeedbackVisibility`? And then call sub methods for each specific survey visibility logic.

Reply:
Yeah, that sounds good to me. I think we won't even need more than one sub-method if [this can be implemented](https://github.com/woocommerce/woocommerce-ios/pull/2657#discussion_r471797585).